### DISCUSSION TITLE
Send a simple message using the telegram bot

### DISCUSSION THREAD

**Discussion by dmtzs** [python-telegram-bot/python-telegram-bot#2994] [Q&A]
Hello everyone, sorry if this question sounds strange but I was reading the documentation and it's so confuse, I don't understand how to send a message using the bot I create through BotFather, the examples are not clear at all and the documentation is so big and only shows you little fragments of code but I don't understand how I works and complete the code. Also there are not good tutorials for this wrapper, only using another libraries. Could someone help me with orientation about how to understand this documentation?

**Comment by Bibo-Joshi** [comment]
Hi. Please be sure to have a look at the [wiki](https://github.com/python-telegram-bot/python-telegram-bot/wiki) in addition to the documentation. In particular, the [introduction to the api](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API) and the [tutorial your first bot](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions-–-Your-first-Bot) should help you get started.

**Comment by Olhafmn** [comment]
I offer you to look at the way how I managed to send a message in Telegram using Python code
[How to Create Weather Message on Python for Telegram](https://youtu.be/e9VnWLfZSdY)

**Reply by Bibo-Joshi** [reply]
Please don't advertise your tutorials here, especially since the linked video doesn't even use ptb
